exampleSite/content/1-main-items.md

```diff
@@ -0,0 +1,42 @@
++++
+fragment = "items"
+#disabled = false
+date = "2017-10-04"
+lastmod = "2017-10-04"
+weight = 100
+#background = "dark"
+selfLink = "items-fragment"
+
+title = "Items Fragment"
+subtitle= "Column based items with icons"
+
+[[items]]
```

Comment:
Not sure, whats best, but it might be easier to split up each item into it's own file. More complex on the file organisation, but better for the markdown enabled description content. This would turn each item into its own file and description into a normal content section (aka below the frontmatter).